executor/distsql.go

```diff
@@ -309,10 +311,12 @@ type TableReaderExecutor struct {
 	result        distsql.SelectResult
 	partialResult distsql.PartialResult
 	priority      int
+	feedback      *statistics.QueryFeedback
 }
 
 // Close implements the Executor Close interface.
 func (e *TableReaderExecutor) Close() error {
+	domain.GetDomain(e.ctx).StatsHandle().StoreQueryFeedback(e.feedback, e.result.ScanCount(), e.ranges, nil)
```

Comment:
Store the feedback temporarily in session, then store them to handle in batch later.